amnestywebsite/humanity-theme
# Replace single.php with FSE template

Ref: #372

**Steps to test**:
1. if applicable, checkout https://github.com/amnestywebsite/wp-plugin-amnesty-branding/pull/46
2. if applicable, checkout https://github.com/amnestywebsite/sharepoint-runner/pull/36
3. checkout this
4. run an asset build
5. view a single post
6. it should not be visibly different to how it would normally appear
7. if applicable, view a document that is translated into multiple languages
8. it should not be visibly different to how it would normally appear
9. switch the document language
10. it should not be visibly different to how it would normally appear
11. if applicable, view a petition
12. it should not be visibly different to how it would normally appear
